src/CMakeLists.txt

```diff
@@ -115,10 +115,6 @@ IF(BUILD_GUI)
    ENDIF(OPENGL_FOUND)
 ENDIF(BUILD_GUI)
 
-#Library: flann
-FIND_PACKAGE(PkgConfig)
```

Comment:
I think you still need a `FIND_PACKAGE(Flann REQUIRED)` or similar. How are the things linked against `libflann` otherwise?